### DISCUSSION TITLE
Can I get Frigate to record Vehicle movements in low light at night?

### DISCUSSION THREAD

**Discussion by andic** [blakeblackshear/frigate#11867] [Ask A Question]
### What is your question:

Hi.
I'm pretty new to Frigate and I'm really loving what I am getting so far. However, I live in a very rural area with no street lighting. I do however get a lot of low cloud (or fog) at night.
My problem is Frigate stops detecting anything after dark and doesn't start again till dawn. I cannot turn on IR illumination in the cameras as that produces movement in the cloud!
Looking at manual recordings, all I see are car headlights and taillights as they disappear, but no matter what I try, Frigate doesn't detect these. 
Any idea how I can get Frigate to detect these 24x7?
I currently have 5 cameras connected to Frigate - the 3 "Gate" cameras are at the entry to my property about 200m away and are the ones I particularly want to detect things in pitch darkness.

My Config:

mqtt:
  enabled: True
  host: 192.168.0.14
  port: 1883
  topic_prefix: frigate
  client_id: frigate

ffmpeg:
  hwaccel_args: preset-rpi-64-h264

audio: # <- enable audio events for all camera
  enabled: false

timestamp_style:
  format: "%d/%m/%Y %H:%M:%S"
  
cameras:
  gate1: 
    ffmpeg:
      inputs:
        - path: rtsp://192.168.0.97:554/11 #The Stream you want to monitor
          roles:
            - record
            - detect
    detect:
      enabled: true # Detection is enabled
      width: 1920 # The Cameras resolution
      height: 1080 # The Cameras resolution
      fps: 5
    motion:
      threshold: 30
      contour_area: 10
      improve_contrast: true
  gate2: 
    ffmpeg:
      inputs:
        - path: rtsp://192.168.0.98:554/11 #The Stream you want to monitor
          roles:
            - record
            - detect
    motion:
      threshold: 30
      contour_area: 100
      improve_contrast: true
  gate3: 
    ffmpeg:
      inputs:
        - path: rtsp://192.168.0.99:554/11 #The Stream you want to monitor
          roles:
            - record
        - path: rtsp://192.168.0.99:554/12 #The Stream you want to monitor
          roles:
            - detect
    detect:
      enabled: true # Detection is enabled
      width: 640 # The Cameras resolution
      height: 352 # The Cameras resolution
      fps: 5
    motion:
      threshold: 30
      contour_area: 10
      improve_contrast: true
  camera1:
    ffmpeg:
      inputs:
      - path: rtsp://192.168.0.91:554/12 #The Stream you want to monitor
        roles:
          - detect
          - record
    detect:
       enabled: true # Detection is enabled
       width: 640 # The Cameras resolution
       height: 352 # The Cameras resolution
       fps: 5
    motion:
      threshold: 255
      contour_area: 10
      improve_contrast: true
  camera2: 
    ffmpeg:
      inputs:
        - path: rtsp://192.168.0.92:554/12 #The Stream you want to monitor
          roles:
            - detect
            - record
    detect:
      enabled: true # Detection is enabled
      width: 1920 # The Cameras resolution
      height: 1080 # The Cameras resolution
      fps: 5
    motion:
      threshold: 255
      contour_area: 10
      improve_contrast: true

database:
  # The path to store the SQLite DB (default: shown below)
  path: /media/frigate/frigate.db

detectors:
  coral:
    type: edgetpu
    device: usb

objects:
  track:
    - person
    - dog
    - bicycle
    - car
    - motorcycle
    - license_plate
    - horse
    - sheep

snapshots:
  enabled: True
  timestamp: true
  bounding_box: true
  retain: 
    default: 30

record:
  enabled: True
  retain:
    days: 3
    mode: motion
  events:
    pre_capture: 5
    post_capture: 5
    retain:
      default: 30
      mode: active_objects
    objects:
    - person
    - dog
    - car
    - bicycle
    - motorcycle
    - license_plate
    - horse
    - sheep
birdseye:
  enabled: True
  width: 800
  height: 600
  quality: 8

**Comment by NickM-27** [comment]
This comes down to a camera problem. I'd suggest enabling IR and just putting a motion mask in the sky.

**Reply by Davy-H** [reply]
I also have a similar issue. I've ended up enabling audio support (with lots of weird sounds enabled) as well to try and help detect vehicles which helps a little.

For example here's a snapshot of a "Roaring Cats"
![image](https://github.com/blakeblackshear/frigate/assets/89611653/567095c1-b68a-4210-8292-630ed7bc4fe6)

Unfortunately as you can see from this snapshot I don't even get a good outline of a car. I think the headlights are just overpower everything.

I've ended up just enabling recording 24/7 on this camera for now till I can figure out a better idea (or camera).
